### DISCUSSION TITLE
Getting Access Token on behalf of User like AWS Cognito

### DISCUSSION THREAD

**Discussion by dhanan-np** [JanssenProject/jans#4909] [Q&A] [answered]
Hello There,

Is there a way we can get access-token of user on behalf user like the AWS Cognito?

**Comment by ossdhaval** [comment]
Adding @yuriyz for comment.

**Comment by yuriyz** [comment, marked as answer]
Custom scripts allow to code own logic for authn/authz. It should be possible to code custom script(s) which will allow it.

**Comment by ossdhaval** [comment]
Hi @dhanan-np As mentioned by @yuriyz, custom script is the tool to achieve what you mentioned. Please go through [custom scripts documentation](https://docs.jans.io/v1.0.13/admin/developer/interception-scripts/) to understand how to use them.
